content/goldcard-profiles.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Gold Card Holders profiles"
++++
+
+
+At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio.
+
+### Art
+
+
+{{< block "grid-2 mt-2 homepage-div" >}}
+
+{{< column >}}
+
+<img style="float:left; padding: 10px" width="200px" height="200px" src="https://www.svgrepo.com/show/79566/avatar.svg" />
```

Comment:
Is it possible to use the markdown syntax or hugo shortcodes  to make this easier for people to edit, and then using CSS to apply the style?